README.md

```diff
@@ -1,2 +1,4 @@
 # wildlife-licencing-tests
 A repository for storing end-to-end tests (For regression testing of the wildlife licensing service, part of SDDS)
+# create .env file from cypress root
+# Add DB_USER=
```

Comment:
How is it built and run?
It will need to register and then use a random user like 781511@email.com for the tests. It cannot rely on finding an empty database and the tests must be repeatable.

Reply:
Tests are currently failing:
`  1) Page title verification
       Verify CD49 - Landowner permission copy-deck:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'View licence applications' within the element: <h1.govuk-heading-xl> but never did.
      at Context.eval (https://new-dev-ing.aws-int.defra.cloud/__cypress/tests?p=cypress/integration/eligibility.js:145:35)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        1                                                                                │
  │ Passing:      0                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     7 seconds                                                                        │
  │ Spec Ran:     eligibility.js                                                                   │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
`